src/analysis/modals/ExportAnalysisModal/useAnalysisExportState.test.ts

```diff
@@ -39,14 +39,12 @@ jest.mock(
   })
 );
 
-type WorkspaceUtilsExports = typeof import('src/workspaces/common/state/useWorkspaces');
-jest.mock(
-  'src/workspaces/common/state/useWorkspaces',
-  (): WorkspaceUtilsExports => ({
-    ...jest.requireActual('src/workspaces/common/state/useWorkspaces'),
+type useWorkspacesExports = typeof import('src/workspaces/common/state/useWorkspaces');
```

Comment:
Just cleanup to match the filename better (and the fact that there's nothing else in `useWorkspaces`).